src/plugin/remark-code-import.ts

```diff
@@ -125,3 +65,61 @@ const remarkIncludeCode = () => {
 };
 
 export default remarkIncludeCode;
+
+function include(indludePath: string, anchor?: string): string {
+  try {
+    let fileContent = fs.readFileSync(indludePath, "utf8");
+
+    // if the anchor is in the format "start:end", extract the lines between the start and end
+    if (anchor && anchor.search(":") != -1) {
+      const [start, end] = anchor.split(":");
+      const lines = fileContent.split("\n");
+      const startIndex = start ? parseInt(start) - 1 : 0;
+      const endIndex = end ? parseInt(end) : lines.length;
+      fileContent = lines.slice(startIndex, endIndex).join("\n");
+    } else if (anchor) {
+      // If an anchor is specified, extract the relevant section from the file between
+      // the start and end anchor comments. This is done using a
+      const commentRegex = rx`
+        \s*         // optional whitespace
+        (/{2,}|#)   // two or more forward slashes or a #
```

Comment:
Do we want one or more `#`?

Reply:
We could - this was mostly for just making it work for showing Cargo.toml. Let's add that when/if we extract this